docs/develop/go/core-application.mdx

```diff
@@ -370,7 +232,9 @@ func YourWorkflowDefinition(ctx workflow.Context, param YourWorkflowParam) (*You
 }
 ```
 
-### How to customize Workflow Type in Go {#customize-workflow-type}
+### Customize Workflow Type in Go {#customize-workflow-type}
```

Comment:
You replaced "in Go" with "using the Go SDK" above so maybe you can replace this with 'using the Go SDK' as well

Reply:
+1 to Angela, who got there before me. 

Also maybe Types instead of Type